src/new-client/src/services/trades/index.ts

```diff
@@ -1,4 +1,5 @@
+import * as tradesTestData from "./__mocks__/mockTrades"
 export * from "./trades"
-export * as tradesTestData from "./__mocks__/mockTrades"
```

Comment:
I had to change this export as safari doesn't support it
https://caniuse.com/mdn-javascript_statements_export_namespace